src/wallet/walletdb.cpp

```diff
@@ -1314,6 +1314,29 @@ DBErrors WalletBatch::ZapSelectTx(std::vector<uint256>& vTxHashIn, std::vector<u
     return DBErrors::LOAD_OK;
 }
 
+bool RunWithinTxn(WalletDatabase& database, const std::string& process_desc, const std::function<bool(WalletBatch&)>& func)
```

Comment:
In commit "wallet: db, introduce 'RunWithinTxn()' helper function" (a8a24cb20974cb8eaec8cb438f6b22489719d605)

Maybe use string_view instead of string, since in most cases this is just passed a string literal, so no need to create a full string object

Reply:
> Maybe use string_view instead of string, since in most cases this is just passed a string literal, so no need to create a full string object

Sure!